tests/test_spec_inspector.py

```diff
@@ -45,12 +47,271 @@ def test_verify_spec_fails_multiple_roles_own_schema_personal_schema(capsys):
                 - jfinance
     """
     spec = yaml.load(spec_yaml)
-    errors = spec_inspector.check_for_multi_schema_owners(spec)
-    expected = spec_inspector.MULTIPLE_SCHEMA_OWNER_ERR_MSG.format('jfinance', 'jfinance, jfauxnance')
+    errors = spec_inspector.ensure_no_schema_owned_twice(spec)
+    expected = spec_inspector.MULTIPLE_SCHEMA_OWNER_ERR_MSG.format('jfinance',
+                                                                   'jfauxnance, jfinance')
     assert [expected] == errors
 
 
-def test_verify_spec_fails_object_referenced_read_write(capsys):
+def test_ensure_no_object_owned_twice(mockdbcontext):
+    mockdbcontext.get_all_object_attributes = lambda: {}
+
+    spec_yaml = """
+    # No config
+    foo:
+
+    # Config but no 'owns'
+    bar:
+        has_personal_schema: True
+
+    # Has 'owns' but not for objkind
+    baz:
+        owns:
+            schemas:
+                - schema0
+
+    role0:
+        owns:
+            tables:
+                - schema0.table0
+                - schema1.table1
+
+    role1:
+        owns:
+            tables:
+                # Ensure function can handle some objects being quoted and some not
+                - schema1.table1
+                - schema1."table2"
```

Comment:
question: are there certain names that **have** to be quoted in order to parse properly?

Reply:
Yeah, if you have a '.' in the table name. For example, there have historically been a few tables that would be like jdoe.jdoe.revenue_metrics. I believe Postgres is actually relatively permissive with table names so long as you throw the name within double quotes.

Reply:
Yep -- this is correct.  You're allowed to do lots of weird things in table/column identifiers as long as you wrap it in double quotes.

```
-- to make everyone uncomfortable when trying to use this in reports
SELECT
*
FROM
cpdean."please never use this it has lots of issues but approximates an important metric";
```